.changeset/unlucky-vans-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos': patch
```

Comment:
This might be a minor version update since we are technically adding some new behavior?